hyp3_gamma/rtc/rtc_sentinel.py

```diff
@@ -409,11 +439,18 @@ def main():
     group.add_argument('--skip-cross-pol', action='store_true',
                        help='Do not include the co-polarization backscatter GeoTIFF in the output package.')
 
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument('--dem', help='Path to the DEM to use for RTC processing. Must be a GeoTIFF in a UTM projection.'
+                                     ' A DEM will be selected automatically if not provided.')
+    group.add_argument('--dem-name', choices=('copernicus', 'legacy'), default='copernicus',
+                       help='DEM to use for RTC processing.')
+
+    parser.add_argument('--bbox', type=float, nargs=4, metavar=('LON_MIN', 'LAT_MIN', 'LON_MAX', 'LAT_MAX'),
+                        help='Subset the output images to the given lat/lon bounding box. Ignored if --dem is '
+                             'provided.')
```

Comment:
Might be a place for [a mutually exclusive group](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group)...

Reply:
`--dem` is exclusive to both `--dem-name` and `--bbox`, but I didn't see a way to enforce that with argparse.  Adding `--bbox` to the exclusive group prevents it from being used together with `--dem-name`.